lab/experiment/__init__.py

```diff
@@ -162,6 +166,10 @@ def calc_configs(self,
             configs_dict = {}
         self.configs_processor = ConfigProcessor(configs, configs_dict)
         self.configs_processor(run_order)
+
+        if configs_dict:
```

Comment:
not sure about this. did this since `configs_dict` are only that one chat change in each iteration.

Reply:
Yeah but if they don't do it with an iterator, the hyper parameters won't be there on Tensboard, isn't it?

Reply:
You are right, let's keep it this way (ignore my previous comment). Because the real use case so far is hyper parameter tuning with a loop.

If we want to log other configs as hyper parameters parameters we can deal with it then,